api/src/main/resources/tanagra.yaml

```diff
@@ -42,13 +41,18 @@ paths:
 ## COMPONENTS
 ##########################################################################################
 components:
+  securitySchemes:
+    oauth:
+      type: apiKey
```

Comment:
QQ: would this change to Bearer token once that is implemented? Or is this app-to-app authorization?

Reply:
OK. I see it in TanagraConfig nevermind :-)